pkg/services/sqlstore/team.go

```diff
@@ -387,8 +387,22 @@ func GetTeamMembers(query *models.GetTeamMembersQuery) error {
 	if query.External {
 		sess.Where("team_member.external=?", dialect.BooleanStr(true))
 	}
-	sess.Cols("team_member.org_id", "team_member.team_id", "team_member.user_id", "user.email", "user.login", "team_member.external", "team_member.permission", "user_auth.auth_module")
-	sess.Asc("user.login", "user.email")
+	sess.Cols(
+		"team_member.org_id",
+		"team_member.team_id",
+		"team_member.user_id",
+		"user.email",
+		"user.name",
+		"user.login",
+		"team_member.external",
+		"team_member.permission",
+		"user_auth.auth_module",
+	)
+	sess.Asc(
```

Comment:
I just noticed that you have changed the sorting.
I think it's better to leave it as it was so as it conforms with the [sorting](https://github.com/grafana/grafana/blob/3f7f128a4fe912f3224cce6b507c0ac30a8a9ef3/pkg/services/sqlstore/org_users.go#L132) in the users table and also the users will have the same experience they used to have.

Reply:
I have now set

```golang
	sess.Asc(
		"user.login",
		"user.email",
		"user.name",
	)
```

I am unsure if this was the expected sorting, or if you wanted to have as previously but w. name added later.

```golang
	sess.Asc("user.email", "user.login")
```

Reply:
> I think it's better to leave it as it was so
I see